scripts/install-windows-prereqs.ps1

```diff
@@ -549,6 +539,23 @@ function Install-VCRuntime {
     }
 }
 
+function Install-AzureDCAPWindows {
```

Comment:
Nice, I like that this is it's own function now

Reply:
Thanks.